### DISCUSSION TITLE
v0.2.3-patch.1

### DISCUSSION THREAD

**Discussion by mojidev-py** [mojidev-py/discmoji#25] [Announcements]
added attachment object and added bunch of other objects
fixed _http and gateway not importing
have not fixed dict error handling yet
**Full Changelog**: https://github.com/mojidev-py/discmoji/compare/v0.2.3...patches

<hr /><em>This discussion was created from the release <a href='https://github.com/mojidev-py/discmoji/releases/tag/patches'>v0.2.3-patch.1</a>.</em>

**Comment by XenonPy** [comment]
Also, @mojidev-py I'm finally back and can continue working on the library :)

**Reply by mojidev-py** [reply]
yayyyy!!!
